codalab/lib/bundle_cli.py

```diff
@@ -1032,11 +1041,13 @@ def do_workers_command(self, args):
                     'worker_id': worker['worker_id'],
                     'cpus': '{}/{}'.format(worker['cpus_in_use'], worker['cpus']),
                     'gpus': '{}/{}'.format(worker['gpus_in_use'], worker['gpus']),
-                    'memory_bytes': formatting.size_str(worker['memory_bytes']),
+                    'memory': formatting.size_str(worker['memory_bytes']),
+                    'free_disk': formatting.size_str(worker['free_disk_bytes']),
                     'last_checkin': '{} ago'.format(
                         formatting.duration_str(int(time.time()) - worker['checkin_time'])
                     ),
                     'tag': worker['tag'],
+                    'running_jobs': ",".join(worker['run_uuids'][0:8]) or "NA",
```

Comment:
- Isn't `worker['run_uuids']` supposed to be a list of uuids?
- I'd remove `NA` and just return `''` if there are no runs - n/a means that it doesn't make sense, but it does here - it just happens to be empty.
- I'd replace `running_jobs` with just `runs` for simplicity.